spec/index.html

```diff
@@ -1718,6 +1716,13 @@ <h3>Parser State</h3>
         The |curObject| is bound to the
         <a href="#grammar-production-object"><code>object</code></a> and
         <a href="#grammar-production-ttObject"><code>ttObject</code></a> productions.</li>
+      <li id="curReifier"><a data-cite="RDF12-CONCEPTS#dfn-rdf-term">RDF Term</a> |curReifier| —
+        The |curReifier| is bound to the
+        <a href="#grammar-production-reifier"><code>reifier</code></a> and
+        <a href="#grammar-production-annotationBlock"><code>annotationBlock</code></a> productions.</li>
+      <li id="curTripleTerm"><a data-cite="RDF12-CONCEPTS#dfn-triple-term">Triple term</a> |curTripleTerm| —
+        Is set in the <a href="#annotation">Annotations</a> constructor.
```

Comment:
Maintaining the earlier description pattern
```suggestion
        The |curTripleTerm| is set in the <a href="#annotation">Annotations</a> constructor.
```